src/main/java/bio/overture/ego/controller/TokenController.java

```diff
@@ -101,6 +110,21 @@ public TokenController(@NonNull TokenService tokenService) {
       @RequestParam(value = "user_id") UUID user_id,
       @RequestParam(value = "scopes") ArrayList<String> scopes,
       @RequestParam(value = "description", required = false) String description) {
+
+    // Check if admin, if not, then check if owner
+    val authentication = SecurityContextHolder.getContext().getAuthentication();
+    if (!authorizationManager.authorizeWithAdminRole(authentication)) {
+      val principal = authentication.getPrincipal();
+      if (principal instanceof User) {
+        User user = (User) principal;
+        if (!user.getId().equals(user_id)) {
+          throw new ForbiddenException("Forbidden");
```

Comment:
Does this message get used? Could be more helpful ("Users cannot create tokens for other users")

Reply:
I was under the impression that in the case a user is performing an action that is `Forbidden` you want to avoid giving them an indication as to the details of why. Could I satisfy you with a log?